Src/Couchbase.Linq/Extensions/QueryExtensions.cs

```diff
@@ -306,7 +307,22 @@ public static IQueryable<T> UseIndex<T>(this IQueryable<T> source, string indexN
         /// <example>
         /// var results = await query.ExecuteAsync();
```

Comment:
If we're updating method summaries, may as well update this one too.

Reply:
Updated to show the cancellationToken parameter on two methods.